docs/source-2.0/aws/aws-endpoints-region.rst

```diff
@@ -282,13 +280,9 @@ Conflicts with
     :ref:`aws.endpoints#standardRegionalEndpoints-trait`
 
 Partitional services (also known as "global" services) resolve a single endpoint per partition.
-That single endpoint is located in the partition's ``defaultGlobalRegion``. Partitional
-services should follow one of two standard patterns:
+That single endpoint is located in the partition's ``defaultGlobalRegion``.
 
-- ``service_dnsSuffix``: ``https://{service}.{dnsSuffix}``
-- ``service_region_dnsSuffix``: ``https://{service}.{region}.{dnsSuffix}``
-
-The following example defines a partitional service that uses ``{service}.{dnsSuffix}``:
+The following example defines a partitional service that uses AWS recommended patterns for each partition:
```

Comment:
Should we provide documentation on what those are?  At least for AWS partition.  We provide a list of the "standard patterns" eg here: https://smithy.io/2.0/aws/aws-endpoints-region.html#aws-endpoints-standardregionalendpoints-trait

Reply:
It's fairly complex though, and currently only captured on an internal w.amazon page, which I don't think is fit for the public docs. If you think we should go the full mile and put the detailed pattern cases in the docs, we can do that though

Reply:
And just adding aws partition is a little misleading since aws partition is itself nonstandard

Reply:
Yeah thats fair - I think its reasonable to leave it unspecified here.

Reply:
If anyone asks, we can presumably link them to the authoritative doc for clarity :)